incubator/hnc/pkg/controllers/setup.go

```diff
@@ -11,10 +12,27 @@ import (
 	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/forest"
 )
 
+var (
+	objectTotal = make(chan int, 100)
+	objectCur   = make(chan int, 100)
+	hierTotal   = make(chan int, 100)
+)
+
 // Create creates all reconcilers.
 //
 // This function is called both from main.go as well as from the integ tests.
 func Create(mgr ctrl.Manager, f *forest.Forest, maxReconciles int, newObjectController bool) error {
+	// Create reconcile counter
+	counter := &ReconcileCounter{
+		Log:         ctrl.Log.WithName("reconcileCounter"),
+		ObjectTotal: 0,
+		ObjectCur:   0,
+		HierTotal:   0,
```

Comment:
Why not have a `HierCur` as well? Not saying you should, just wondering why we don't.

If you did, we could just have a single `updateReconcileCounter` function across both (temporarily: all three) controllers that just indicated which totals we were updating.

Reply:
I added it now just to be more informative. (Added as atomic counter)